### ISSUE TITLE
Add --no-multiarch when installing

### ISSUE THREAD

**Issue by hetong007** [jeremiedb/mxnet_winbin#3]
In the last part of the instruction, would it be better to execute `R CMD INSTALL --no-multiarch R-package`?

**Comment by jeremiedb** [comment]
Yes I think it makes sense. I guess it didn't cause me issue as I'm only running 64 bit R. 
And AFAIK, 32 bit shouldn't be working anyway.
